cypress/integration/basic/role_assignment_made.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Test that role assignments can be made', () => {
+  const BOOKMARKED_PERSON = 'Sadaf Arshad';
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_DSS-RM_session');
```

Comment:
Make the string a constant in case Rails changes it's cookie name in the future.